pkg/api/validate/plugin.go

```diff
@@ -53,6 +55,19 @@ func (v *PluginAPIValidator) Validate(t *pluginapi.Config) (errs []error) {
 	return errs
 }
 
+func (v *PluginAPIValidator) validatePluginTemplateComponentLogLevels(c *pluginapi.ComponentsLogLevel) (errs []error) {
+	if c.ApiServer%2 != 0 || (c.ApiServer < 0 || c.ApiServer > 8) {
```

Comment:
the %2 does not make sense.  Make the types uints, then check they are not larger than, say, 20.

Reply:
valid logging levels are 0,2,4,6 and 8. The %2 checks that the supplied log level is an even number. Am i missing something?